.github/dot/specs.dot

```diff
@@ -73,4 +73,5 @@ graph {
         Semigroup -- Arrow_alt [style=dotted; arrowhead=none];
 
         Meet_semilattice -- Semigroup [style=dotted; arrowhead=none; constraint=false];
+        Join_semilattice -- Semigroup [style=dotted; arrowhead=none; constraint=false];
```

Comment:
I suggest to add the same relation betweenn join and meet

Reply:
done, tell me if its ok now